partials/anunt.hbs

```diff
@@ -0,0 +1,75 @@
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+
+<!-- Carousel -->
+<style>
+    .container-carousel {
```

Comment:
After fixing the classes from boostrap, if you delete de subtitle, the carrousel has no space between it and the title; Use padding or margin to solve this